src/main/resources/templates/languages/en/non-strategic/sscsMidlandsDailyHearingList.json

```diff
@@ -0,0 +1,17 @@
+{
+  "listName": "Midlands First-tier Tribunal (Social Security and Child Support) Daily Hearing List",
+  "openJusticeStatement": "Open justice is a fundamental principle of our justice system. When considering the use of telephone and video technology, the judiciary will have regard to the principles of open justice. Judges may determine that a hearing should be held in private if this is necessary to secure the proper administration of justice.",
+  "contactMessage": "Social Security and Child Support Tribunal parties and representatives will be informed directly as to the arrangements for hearing cases remotely. Any other person interested in joining the hearing remotely should contact the Social Security and Child Support Tribunal Office direct, in advance of the hearing date, by emailing EMAIL so that arrangements can be made. The following details should be included in the subject line of the email [OBSERVER/MEDIA] REQUEST – [case reference] – [hearing date]. If the case is to be heard in private or is subject to a reporting restriction, this will be notified.",
+  "emailAddress": "ascbirmingham@justice.gov.uk",
+  "tableHeaders": [
+    "Venue",
+    "Appeal Reference Number",
```

Comment:
For all other non-strategic list types, only the first letter of the first word is in upper case. The rest is in lower case like "Appeal reference number". It is better to keep it consistent.

Reply:
fixed.